subworkflows/nf-scil/topup_eddy/main.nf

```diff
@@ -0,0 +1,60 @@
+// ** Importing modules from nf-scil ** //
+include { PREPROC_TOPUP } from '../../../modules/nf-scil/preproc/topup/main'
+include { PREPROC_EDDY } from '../../../modules/nf-scil/preproc/eddy/main'
+include { UTILS_EXTRACTB0 } from '../../../modules/nf-scil/utils/extractb0/main'
+
+workflow TOPUP_EDDY {
+
+    // ** The subworkflow will run topup if a reverse b0 or reverse DWI is provided ** //
+    // ** the EDDY and finally it will also extract a b0 from corrected DWI         ** //
```

Comment:
```suggestion
    // ** The subworkflow will optionally run topup if a reverse b0 or reverse DWI is provided.       ** //
    // ** In both cases, it will perform EDDY and also extract a b0 from the corrected DWI image. ** //
```